common/lib/xmodule/xmodule/js/spec/video/video_control_spec.js

```diff
@@ -278,7 +278,8 @@
 
         describe('constructor with end-time', function () {
             it(
-                'saved position is 0, timer slider and VCR set to 0:00',
+                'saved position is 0, timer slider and VCR set to 0:00 ' + 
```

Comment:
Is there a reason why you modify existing tests, instead of creating new ones?
(Maybe the answer is that this version of the test still tests the previous scenario?)